content/_data/changelogs/weekly.yml

```diff
@@ -10070,6 +10070,57 @@
   # pull: 5183 (PR title: Ignore old servlet api in dependabot)
   # pull: 5185 (PR title: Bump Mailer from 1.23 to 1.32.1)
 
+- version: '2.277'
+  date: 2021-01-26
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 5176
+    authors:
+    - fqueiruga
+    - timja
+    message: |-
+      Use a more accessible color palette in configuration form tabs.
+  - type: rfe
+    category: rfe
+    pull: 5198
+    authors:
+    - res0nance
+    message: |-
+      Improve performance when saving fingerprints to disk.
+  - type: rfe
+    category: rfe
+    pull: 5190
+    issue: 64670
+    authors:
+    - alecharp
+    - batmat
+    message: |-
+      Improve performance in fingerprint file creation.
```

Comment:
I would rather merge this entry with one above. Actually it is also about saving any fingerprint, not just creation

Reply:
I like that idea.  It is not common that we have two pull requests addressing the same area, but a reference that links to both pull requests and including all authors in the same entry seems like the best thing to do for a user focused changelog.

Reply:
See 017e6e7a7a1dd87a5ad916bcb1de0a375c5a1fd2